presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -78,6 +78,14 @@ Spilling Properties
     aggregations and joins (inner and outer), so this property will not reduce memory usage required for
     window functions, sorting and other join types.
 
+``optimizer.reorder-windows``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Boolean``
+ * **Default value:** ``true``
+ * **Description:** Allow reordering windows in order to put those with the same partitioning next to each other. This will sometimes decrease the number of repartitionings. This can also be specified on a per-query basis using the ``reorder_windows`` session property.
```

Comment:
to me the word `sometimes` means it very rare case. I would prefer the word `may` as it means it is possible. A bit more positive. 

I would say something about possible benefits and costs.